services/voting-node/development.md

```diff
@@ -0,0 +1,61 @@
+# Voting Node Service
+
+This service acts as a wrapper around a `jormungandr`, interacting with the Event DB to gather the necessary so that the node can join in the network.
```

Comment:
```suggestion
This service acts as a wrapper around a `jormungandr`, interacting with the Event DB to gather the necessary data so that the node can join in the network.
```